docs/guide/database-insert-update.md

```diff
@@ -0,0 +1,159 @@
+        // Insert the dog with the asynchronous `self.insert` method
+        const insertResult = await self.insert(req, newDog);
+
+        return insertResult;
+      }
+    }
+  };
+  ```
+  <template v-slot:caption>
+    modules/dog/index.js
+  </template>
+</AposCodeBlock>
+
+## Inserting pages
+
+As mentioned above, pages have a position within the page tree. The `insert` method for pages therefore requires additional arguments to place a new page in that hierarchy properly. Also [as described above](#updating-page-documents) we normally call the insert method directly from the `@apostrophecms/page` module, aliased in server-side code as `self.apos.page`.
+
+The arguments for the `self.apos.page.insert()` method are:
+
+| Argument | Required | Description |
+| -------- | -------- | ----------- |
+| `req` | TRUE | The associated request object. Using a provided `req` object is important for maintaining user role permissions. |
+|`_targetId` | TRUE | The `_id` of an existing page to use as a target when inserting the new page. `_home` and `_archive` are optional conveniences for the home page and [archived section](/reference/api/pages.md#moving-pages-to-the-archive), respectively. |
+|`_position` | TRUE | A numeric value will represent the zero-based child index under the `_targetId` page. `before`, `after`, `firstChild`, or `lastChild` values set the position within the page tree for the new page in relation to the target page (see `_targetId`). `before` and `after` insert the new page as a sibling of the target. `firstChild` and `lastChild` insert the new page as a child of the target. |
+| `page` | TRUE | The page document object. |
+| `options` | FALSE | An options object, primarily used for internal draft state management. |
+
+As with pieces, this process will normally begin by generating an empty page document with the `newInstance()` method, which should be done using a specific page type module.
+
+
+<AposCodeBlock>
+  ```javascript
+  let newPage = self.newInstance();
+
+  newPage = {
+    ...newPage,
+    ...initialInfo
+  };
+
+  await self.apos.page.insert(req, '_home', 'lastChild', newPage);
+  ```
+  <template v-slot:caption>
+    modules/special-page/index.js
+  </template>
+</AposCodeBlock>
+
+::: note
+Whenever updating or inserting a content document, do not attempt to directly set any property beginning with an underscore (`_`). Any property with a leading underscore, except for `_id`, **will not be saved to the database**.
+
+Apostrophe only allows starting property names with an underscore on dynamically loaded information. This includes [populated relationship information](/guide/relationships.md) and the `_url` property. These properties can change at any time and make use of other information stored elsewhere in the database.
+
+With very few exceptions, the `_id` property is an automatically generated, randomized, and (always) unique property. It is also special in that it can never change for a given document.
```

Comment:
Correct. You should not try to set it yourself because you're inviting problems with Apostrophe's modes and locales if you do.